src/Features/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptCompletionServiceWithProviders.cs

```diff
@@ -17,5 +21,23 @@ internal sealed override CompletionRules GetRules(CompletionOptions options)
             => GetRulesImpl();
 
         internal abstract CompletionRules GetRulesImpl();
+
+        internal sealed override void FilterItems(
```

Comment:
This is added so we can move TS to use the internal overload instead, which will provide some perf benefit plus we can then make the public overload non-virtual.